### PR TITLE
Fix a few crashes on 1.9

### PR DESCRIPTION
**Pull request by jb55**
Some japanese user profiles are breaking the flatbuffer profile builder for some reason

Changelog-Fixed: Fix pretty bad crash when building flatbuffer profiles

### CODE REVIEW

**Comment by jb55** [comment]
Went through and fixed a bunch more crashes

File: nostrdb/flatcc/flatcc_builder.h [hunk 1]
```diff
@@ -82,7 +82,7 @@ extern "C" {
  * Note: some internal assertion will remain if disabled.
  */
 #ifndef FLATCC_BUILDER_ASSERT_ON_ERROR
-#define FLATCC_BUILDER_ASSERT_ON_ERROR 1
+#define FLATCC_BUILDER_ASSERT_ON_ERROR 0
```
Comment (danieldaquino) [inline nostrdb/flatcc/flatcc_builder.h:85]: Have we ruled out the risk that the aforementioned errors could corrupt flatbuffer data and cause further issues down the line?

File: damus/Models/MutelistManager.swift
Comment (danieldaquino) [inline damus/Models/MutelistManager.swift:1]: The changes in this file LGTM

File: nostrdb/flatcc/flatcc_builder.h (hunk 1 above)
Comment (jb55) [reply nostrdb/flatcc/flatcc_builder.h:85]: it is handled here:

```
	NdbProfile_parse_json_table(ctx, buf, buf + bufsiz, profile);
	if (ctx->error)
		return 0;
```

in `ndbprofile_parse_json`

**Comment by jb55** [comment]
Closing in favor of #2336

File: nostrdb/flatcc/flatcc_builder.h (hunk 1 above)
Comment (danieldaquino) [reply nostrdb/flatcc/flatcc_builder.h:85]: Nice!

Do we have similar guards around other flatbuffer objects like ndbnotes and such (given that this constant looks global)? If we do, then this LGTM

**Pull request closed without merging**